functorch/csrc/BatchRulesRandomness.cpp

```diff
@@ -149,6 +149,33 @@ Tensor unary_pointwise_random_batch_rule(const Tensor& tensor, ExtraArgs... extr
   return makeBatched(out, 0, cur_level);
 }
 
+template<typename... ExtraArgs>
+Tensor randint_like_random_batch_rule(const Tensor& self, ExtraArgs... extra_args) {
```

Comment:
@samdow -- how do we handle randn_like, rand_like right now?

Reply:
Good point--didn't see those in the original randomness error message but I'll double check them and probably end up adding them here